build/rsi/run/README.md

```diff
@@ -0,0 +1,8 @@
+# Run RSI Docker image locally
```

Comment:
I think `run` folder should not be inside `build` as it's an "application" of the docker image not how to build it.

That said, maybe give it a more specific name other than "run"? Same apply for "run_docker.sh" which is too generic.

Reply:
Any thoughts on where the folder should reside? We can give it the same name that we come up with for the build folder and call it `run_that_name.sh`.

One option is to put it under a new `tools` folder. And suppose we go with `build/simple` for the build folder, this can be called `tools/simple` and the run file, `run_simple.sh`.

The full set of files will look like this:

`import/simple` (code)
`import/build/simple/Dockerfile`
`import/tools/simple/run_simple.sh`
`import/build/ci/cloudbuild.simple.yaml`

etc.

Reply:
I like this idea!

Reply:
Done.